### DISCUSSION TITLE
不能做个教授視頻吗

### DISCUSSION THREAD

**Discussion by pannasiri** [MuiseDestiny/zotero-gpt#255] [Ideas]
可是 我看字不太懂，您能不能做个教授視頻吗。因为我想回用 Zotero-gpt，但是不太懂英文的意思呀。

**Comment by cziyo** [comment]
https://www.bilibili.com/video/BV17N4y1o7vx/?spm_id_from=333.337.search-card.all.click&vd_source=e0fda384114ed6bac3c735bbc22d91b9 b站上有视频
